IdentityCore/tests/integration/MSIDAuthorityIntegrationTests.m

```diff
@@ -568,7 +568,7 @@ - (void)testDiscoverAuthority_whenValidOnPremADFSAuthorityValidateYes_shouldRetu
     [MSIDTestURLSession addResponse:response];
 
     // Web finger response.
-    __auto_type webFingerRequestUrl = [@"https://example.com/.well-known/webfinger?resource=https://login.windows.com/adfs" msidUrl];
+    __auto_type webFingerRequestUrl = [@"https://example.com/.well-known/webfinger?resource=https%3A%2F%2Flogin.windows.com%2Fadfs" msidUrl];
```

Comment:
Not sure why this changed. Theoretically, there should be no differences in any tests outside of space cases. Just adding a comment to double check that change was intentional.

Reply:
Previous test is incorrect. it wasn't even a valid URL before,